syft/format/internal/cyclonedxutil/helpers/licenses.go

```diff
@@ -5,11 +5,15 @@ import (
 	"strings"
 
 	"github.com/CycloneDX/cyclonedx-go"
-
 	"github.com/anchore/syft/internal/spdxlicense"
 	"github.com/anchore/syft/syft/pkg"
 )
 
+const (
+	noAssertion     = "NOASSERTION"
```

Comment:
CycloneDX should not use NOASSERTION, this is SPDX specific. CycloneDX should just leave this field empty if no copyright is found

Reply:
גםמק